### PR TITLE
fix: helm release 12.1 for default settlement functionality 

### PR DESCRIPTION
**Pull request by vgenev**
* Added new column settlementModelId to settlementWindowContent as foreign key to settlementModel

### CODE REVIEW

File: migrations/950107_settlementWindowContent_foreign_settlementModel.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              t.foreign('SettlementModelId').references('SettlementModelId').inTable('settlementModel')
+            })
+          }
+        })
+    }
+  })
+}
+
+exports.down = function (knex) {
+  return knex.schema.dropTableIfExists('settlementWindowContent')
```
Comment (mdebarros) [inline migrations/950107_settlementWindowContent_foreign_settlementModel.js:45]: This should remove the key constraint and the column, not drop the table.

**Review by mdebarros — CHANGES_REQUESTED** [review]
See my comment.

File: migrations/950107_settlementWindowContent_foreign_settlementModel.js (hunk 1 above)
Comment (vgenev) [reply migrations/950107_settlementWindowContent_foreign_settlementModel.js:45]: oh.. ok..

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**